### DISCUSSION TITLE
Why is BagPart empty after passing ContentItem to method as param or querying?

### DISCUSSION THREAD

**Discussion by SzymonSel** [OrchardCMS/OrchardCore#15732] [Q&A]
I have a ContentItem having a named BagPart as such: contentItem.Content.Items.Contentems.

I update the the contentItem  by contentItem by
````
contentItem.Apply("Items", new BagPart { ContentItems = proposalItems });

await _contentManager.UpdateAsync(contentItem);
````

This works as expected, but after upgrading our code to OC 1.9.* `contentItem.Content.Items == null` if we pass the object to another methos or simply do a `await _contentManager.GetAsync(contentItem.ContentItemId)`.... unless we get the items by calling `contentItem.Get<BagPart>("Items")`.

Why so? Where is the Items reference gettting lost in the same call stack?

**Comment by hishamco** [comment]
@MikeAlhayek did you work on something related recently?
